.zenodo.json

```diff
@@ -0,0 +1,120 @@
+{
+    "creators": [
+      {
+        "affiliation": "HZDR, TU Dresden",
+        "name": "Huebl, Axel",
+        "orcid": "0000-0003-1943-7141"
+      },
+      {
+        "affiliation": "HZDR",
+        "name": "Widera, René",
+        "orcid": "0000-0003-1642-0459"
+      },
+      {
+        "affiliation": "LogMeIn, Inc.",
+        "name": "Worpitz, Benjamin"
+      },
+      {
+        "affiliation": "HZDR, TU Dresden",
+        "name": "Pausch, Richard",
+        "orcid": "0000-0001-7990-9564"
+      },
+      {
+        "affiliation": "HZDR, TU Dresden",
```

Comment:
isn't his current affiliation: Max Planck Institute for the Physics of Complex Systems

see: https://www.pks.mpg.de/dynamics-in-correlated-quantum-matter/people/heiko-burau/

Reply:
I just exported the current authors for now, thanks for the hint! When he contributed the work for the last release, that was his affiliation. We usually would then just update the affiliation for recurring authors over time.

Even if he does not actively contribute anymore, @Heikman contributed a significant part to the codebase that will obviously conserve his co-authorship. In this specific case, it could therefore be a good compromise to keep old and new affiliations at the time to reflect that corner-case.

Reply:
@ax3l So do you want to update @Heikman 's affiliation **now** - or **later**?

Reply:
if he doesn't respond currently, we merge it as it for now.